<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvanceSettingsDialog</class>
 <widget class="QDialog" name="AdvanceSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Advance Settings</string>
  </property>
  <widget class="QFrame" name="editFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="socks5AddressLabel">
      <property name="text">
       <string>Local Socks5 Address</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="socks5Address"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="socks5PortLabel">
      <property name="text">
       <string>Local Socks5 Port</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="socks5Port"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="httpAddressLabel">
      <property name="text">
       <string>Local Http Address</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="httpAddress">
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="httpPortLabel">
      <property name="text">
       <string>Local Http Port</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="httpPort">
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="pacAddressLabel">
      <property name="text">
       <string>Local PAC Address</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="pacAddress"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="pacPortLabel">
      <property name="text">
       <string>Local PAC Port</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="pacPort">
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly|Qt::ImhPreferNumbers</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="enableHttpMpdeLabel">
      <property name="text">
       <string>Enable Http Mode</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="httpModeCheckBox">
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="logLevelLabel">
      <property name="text">
       <string>Log Level</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="logLevelComboBox">
      <item>
       <property name="text">
        <string>ALL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>INFO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WARN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ERROR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FATAL</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OFF</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AdvanceSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
